packages/material-ui/src/Popper/Popper.js

```diff
@@ -210,11 +222,7 @@ const Popper = React.forwardRef(function Popper(props, ref) {
         role="tooltip"
         {...other}
         style={{
-          // Prevents scroll issue, waiting for Popper.js to add this style once initiated.
-          position: 'fixed',
-          // Fix Popper.js display issue
-          top: 0,
-          left: 0,
```

Comment:
Can't we just set `display: none` here conditionally if `!open && keepMounted`? The current approach makes it harder to track which state does what.

Reply:
I had thinking about it initially
It really gets better